Libplanet/Net/Swarm.cs

```diff
@@ -1677,6 +1656,41 @@ private async Task RepeatReplyAsync(CancellationToken token)
             }
         }
 
+        private async Task RepeatBroadcastAsync(CancellationToken token)
+        {
+            TimeSpan interval = TimeSpan.FromMilliseconds(100);
+            while (Running)
+            {
+                if (_broadcastQueue.TryDequeue(out Message raw, interval))
+                {
+                    NetMQMessage message = raw.ToNetMQMessage(_privateKey);
+
+                    // FIXME Should replace with PUB/SUB model.
+                    try
+                    {
+                        await Task.WhenAll(
+                            _dealers.Values.Select(
+                                s =>
+                                Task.Run(() => s.SendMultipartMessage(message))
+                            )
+                        );
+                    }
+                    catch (TimeoutException e)
+                    {
+                        _logger.Error(e, "TimeoutException occured.");
+                    }
+                    catch (Exception e)
+                    {
+                        _logger.Error(e, "UnexpectedException occured.");
```

Comment:
This looks like a name of exception type is `UnexpectedException`.

```suggestion
                        _logger.Error(e, "An unexpected exception occured.");
```